custom/conf/app.example.ini

```diff
@@ -1046,3 +1046,29 @@ QUEUE_CONN_STR = "addrs=127.0.0.1:6379 db=0"
 MAX_ATTEMPTS = 3
 ; Backoff time per http/https request retry (seconds)
 RETRY_BACKOFF = 3
+
+; default storage for attachments, lfs and avatars
+[storage]
+; storage type
+STORE_TYPE = local
```

Comment:
Would call this `STORAGE_NAME` because type is confusing when user can choose a name.

Reply:
It has been used by previous PRs and I think the change will break them.

Reply:
Was it already commited? If not I see no reason not to rename now. Otherwise add a comment that this variable refers to user-defined names below.

Reply:
This was only committed very recently and not in any releases right? I don't think that is a real breaking change then and do think it would save confusion in the actual release

Reply:
So `STORAGE_NAME` or `STORAGE_TYPE` which is better?

Reply:
I'd suggest `storage.STORAGE_NAME` and `storage.myname.STORAGE_TYPE`.

Reply:
@silverwind this is referring to the backend model used to run the storage, e.g. either minio, local file storage, some other type yet to be determined... 

This particular definition says that the default storage backend will be local filestorage not something else

Reply:
Right, in that case "type" is fine. So I suggest just changing to `storage.STORAGE_NAME`.

Reply:
In fact, the name is `default` for `[storage]` and `xxxx` for `[storage.xxx]`. So I think `STORAGE_TYPE` here is better.

Reply:
Maybe a more detailed comment above that explains better what this value is? @zeripath description would be better